charts/psmdb-db/production-values.yaml

```diff
@@ -0,0 +1,130 @@
+# Default values for pxc-cluster.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# Platform type: kubernetes, openshift
+# platform: kubernetes
+
+# Cluster DNS Suffix
+# DNSsuffix: .svc.cluster.local
+
+nameOverride: ""
+fullnameOverride: ""
+
+pause: false
+allowUnsafeConfigurations: false
+# updateStrategy: SmartUpdate
```

Comment:
smart update is enabled by default in the official image, should we enable it here also?
and we should probably add more fields (even if they are commented): https://github.com/percona/percona-server-mongodb-operator/blob/master/deploy/cr.yaml#L14-L18

Reply:
done